### ISSUE TITLE
Allow multiple patterns

### ISSUE THREAD

**Issue by rubennorte** [tomchentw/unused-files-webpack-plugin#2]
Right now we can only use a single pattern to detect unused files:

``` javascript
new UnusedFilesWebpackPlugin({
  pattern: 'js/src/**'
})
```

It would be nice if we could set an array of patterns. That way we could check if all files in src and certain files in the test directory, for example, are used or not:

``` javascript
new UnusedFilesWebpackPlugin({
  patterns: [
    'js/src/**',
    'js/test/support/**',
    'js/test/specs/**'
})
```

I think it would be as easy as replace `glob` with `multi-glob`

**Comment by tomchentw** [comment]
Good idea!

**Comment by mattdell** [comment]
Agreed it would be much easier, but if you're stuck with this you can still do it with the current syntax:

In my solution I've done this:
```
new UnusedFilesWebpackPlugin({
	pattern: '+(lib|assets)/**/*.*'
}),
```

See syntax here: https://github.com/isaacs/node-glob#glob-primer

**Comment by tomchentw** [comment]
Any suggested libraries for multiple glob patterns?

**Comment by QWp6t** [comment]
@tomchentw `glob-all` - https://www.npmjs.com/package/glob-all

In a similar plugin I wrote, I was able to use `glob-all` as a drop-in replacement for `glob`. I think it also accepts both strings and arrays. You could try it and see if it passes your current tests.

**Comment by tomchentw** [comment]
@QWp6t thanks! This looks interesting

**Comment by tomchentw** [comment]
Released [v3.2.0](https://github.com/tomchentw/unused-files-webpack-plugin/releases/tag/v3.2.0)
